packages/neovim/src/utils/getNvimFromEnv.test.ts

```diff
@@ -3,42 +3,40 @@
 import which from 'which';
 import {
   getNvimFromEnv,
-  compareVersions,
-  parseVersion,
+  exportsForTesting,
 } from './getNvimFromEnv';
+const parseVersion = exportsForTesting.parseVersion
+const compareVersions = exportsForTesting.compareVersions
```

Comment:
Make these functions non-exported. If `nvim` is found then clients can use Nvim's own `vim.version` module, so there is no need for node-client to expose a half-baked "semver" implementation.